mock_arduino.py

```diff
@@ -1,2 +1,10 @@
-def mock_arduino_response():
-    print("Your command did something!")
+from typing import Text
+
+
+def mock_arduino_response(text):
+    if text == "ledon":
+        return str("ledon")
```

Comment:
You don't need to convert these strings to strings, they already be strings :D (if the conversion needs to happen for the reply from a real arduino, that will happen in the serial file, so we don't need to worry about it here).

Reply:
Nice! done